apps/token-manager/app/src/components/AutoComplete/AutoCompleteSelected.js

```diff
@@ -0,0 +1,108 @@
+import React, { useRef, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { ButtonBase, theme } from '@aragon/ui'
+import AutoComplete from './AutoComplete'
+
+const identity = x => x
+const noop = () => null
+
+const AutoCompleteSelected = React.memo(
+  React.forwardRef(function AutoCompleteSelected(
+    {
+      itemButtonStyles,
+      items,
+      onChange,
+      onSelect,
+      onSelectedClick = noop,
```

Comment:
Can you add a comment about the difference between `onSelect` and `onSelectedClick`. 

I'm not sure I got it based on the usage below.

Reply:
Sure.

Reply:
Reference:
`onSelect` happens when the user clicks on an item (selecting the item).
`onSelectedClick` happens when the user clicks on the rendered selected item.